our-brothers-web/package.json

```diff
@@ -27,8 +27,10 @@
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
     "@fortawesome/fontawesome-free": "^5.12.1",
+    "aos": "^2.3.4",
```

Comment:
Please change to `~2.3.4` and also change `@fortawesome/fontawesome-free` to `~5.12.1`

Reply:
fixed